resources/lang/pl/beatmapsets.php

```diff
@@ -50,13 +50,14 @@
             ],
             'favourite' => 'Dodaj tę beatmapę do ulubionych',
             'unfavourite' => 'Usuń tę beatmapę z ulubionych',
+            'favourited_count' => '+ 1 inna!|+ :count inne!|+ :count innych!',
         ],
         'stats' => [
             'cs' => 'Wielkość kółek',
             'cs-mania' => 'Ilość klawiszy',
             'drain' => 'Spadek HP',
             'accuracy' => 'Celność',
-            'ar' => 'Approach Rate',
+            'ar' => 'Prędkość otoczki',
```

Comment:
this sounds strange? is this how it's translated everywhere else?

Reply:
no tak galkan kazał 🤔

Reply:
oh

Reply:
można jeszcze „obręczy” dać, jeżeli wam to brzmi lepiej; bo na pewno nie APPROACH CIRCLE'A